docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -32,6 +32,7 @@ for instance:
     - this script serves as a fast way to install demo instance of Shopsys Framework.
     - also this script can be used if you change the configuration of docker or app, script will apply all the changes done in these files and rebuild images.
 - add a way to check if Redis is running [#815](https://github.com/shopsys/shopsys/pull/815)
+    - change version of snc/redis-bundle to ^2.1.8 in your composer.json and update dependencies with `composer update`
```

Comment:
i am not sure that this should be done before the recreation of container but we`ll see during testing or upgrade